src/components/Score.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Score(props) {
+  const { answers } = props;
+  const correctAnswers = answers.filter((answer) => answer.correct);
```

Comment:
this value should be numerical sent by the parent to this

Reply:
Done